test/modules/core/effects/lighting/sunlight.spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable */
+import test from 'tape-catch';
+import {getSolarPosition} from '@deck.gl/core/effects/lighting/sunlight';
+
+const MS_A_HOUR = 3.6e6;
+
+function almostEqual(v1, v2, epsilon = 0.3) {
+  return Math.abs(v1 - v2) < epsilon;
+}
+
+test('Sunlight#azimuth and altitude', t => {
+  const latitude = 49.253;
+  const longitude = -123.13;
+
+  let timestamp = 1553990400000; // 03/31/2019 @ 12:00am (UTC)
+  timestamp += 7 * MS_A_HOUR; // Vancouver GMT-7
+
+  // azimuth and altitude values are calculated from
+  // https://www.wolframalpha.com/input/?i=solar+position+on+20:00+03-31-2019+at+Vancouver
+  const testCases = [
```

Comment:
Nit: CAPITAL_CASE for constants. Maybe move test cases constant out of the function?